llvm/test/Verifier/convergence-laneops-xfail.ll

```diff
@@ -0,0 +1,19 @@
+; RUN: not --crash llc -stop-after=amdgpu-isel -mtriple=amdgcn-- -mcpu=gfx900 -verify-machineinstrs -o - %s 2>&1 | FileCheck %s
```

Comment:
This is not an IR verifier test, it is a codegen test that fails the machine verifier. A machine verifier would go in test/MachineVerifier, and preferably would be written in MIR and not rely on codegen.


I thought the point of this test was to show that the selection did not produce a machine verifier error after selection, so this is broken

Reply:
This should also be repeated for all 3 intrinsics